site/installing.md

```diff
@@ -11,6 +11,7 @@ Weave Cloud is the recommended option if:
  * You are deploying to larger clusters.
  * You require secure remote access.
  * You want to share access with your coworkers.
+ * You want to minimize Weave Scope memory and CPU usage.
```

Comment:
But it's the memory and CPU usage of *Scope* that users of the standalone Scope observe as being high.